docs/source/whatsnew_1_4.md

```diff
@@ -0,0 +1,68 @@
+
+
+## MAISI: state-of-the-art 3D Latent Diffusion Model
+
+![maisi](../images/maisi_train.png)
+
+MAISI (Medical AI for Synthetic Imaging) is a state-of-the-art three-dimensional (3D) Latent Diffusion Model designed for generating high-quality synthetic CT images with or without anatomical annotations. This AI model excels in data augmentation and creating realistic medical imaging data to supplement limited datasets due to privacy concerns or rare conditions. It can also significantly enhance the performance of other medical imaging AI models by generating diverse and realistic training data.
+
+A tutorial for generating large CT images accompanied by corresponding segmentation masks using MAISI is provided within
+[`project-monai/tutorials`](https://github.com/Project-MONAI/tutorials/blob/main/generation/maisi).
+And it contains the following features:
+- A foundation Variational Auto-Encoder (VAE) model for latent feature compression that works for both CT and MRI with flexible volume size and voxel size
+- A foundation Diffusion model that can generate large CT volumes up to 512 × 512 × 768 size, with flexible volume size and voxel size
+- A ControlNet to generate image/mask pairs that can improve downstream tasks, with controllable organ/tumor size
+
+## VISTA-3D: state-of-the-art 3D Latent Diffusion Model
+
+![vista-3d](../images/vista3d.png)
+
+VISTA-3D is a specialized interactive foundation model for 3D medical imaging. It excels in providing accurate and adaptable segmentation analysis across anatomies and modalities. Utilizing a multi-head architecture, VISTA-3D adapts to varying conditions and anatomical areas, helping guide users' annotation workflow.
+
+A tutorial show how to finetune VISTA-3D on spleen dataset is provided within
+[`project-monai/tutorials`](https://github.com/Project-MONAI/tutorials/blob/main/vista_3d).
+It can support three core workflows:
+- Segment everything: Enables whole body exploration, crucial for understanding complex diseases affecting multiple organs and for holistic treatment planning.
+- Segment using class: Provides detailed sectional views based on specific classes, essential for targeted disease analysis or organ mapping, such as tumor identification in critical organs.
+- Segment point prompts: Enhances segmentation precision through user-directed, click-based selection. This interactive approach accelerates the creation of accurate ground-truth data, essential in medical imaging analysis.
+
+## VISTA-2D: cell segmentation pipeline
+
+![vista-2d](../images/vista2d.png)
+
+VISTA-2D is a comprehensive training and inference pipeline for cell segmentation in imaging applications. For more information, refer to this [Blog](https://developer.nvidia.com/blog/advancing-cell-segmentation-and-morphology-analysis-with-nvidia-ai-foundation-model-vista-2d/)
+
+Key features of the model include:
+- A robust deep learning algorithm utilizing transformers
+- Generalist model as compared to specialist models
```

Comment:
Maybe "general-purpose" instead of generalist, though I'm not sure what's being said here. Is this not a foundational model then?

Reply:
Yes, it's a foundational model for cell segmentation and actually I refer to the highlights mentioned in this blog: https://developer.nvidia.com/blog/advancing-cell-segmentation-and-morphology-analysis-with-nvidia-ai-foundation-model-vista-2d/

Reply:
I'd call it foundational here then to highlight the contribution.

Reply:
Updated, thanks.